src/dataflow/src/source/mod.rs

```diff
@@ -237,6 +238,7 @@ pub(crate) trait SourceConstructor<Out> {
         worker_id: usize,
         worker_count: usize,
         logger: Option<Logger>,
+        rdkafka_logger: Option<Logger>,
```

Comment:
This is turning into a _ton_ of arguments, wow. Sounds like @benesch's suggestion will cut that argument? Otherwise, I guess one day someone should make a constructor with fewer muddled responsibilities (:

Reply:
Yes, this will be removed in the new patchset.